src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -828,11 +852,61 @@ public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
                   },
                   ""frames"": [
                     {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          },
+                          ""region"": {
+                            ""startLine"": 15,
+                            ""startColumn"": 9
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Evaluate-0"",
+                        ""message"": {
+                          ""text"": ""Exception thrown""
+                        }
+                      },
                       ""module"": ""RuleLibrary"",
                       ""threadId"": 52,
                       ""address"": 10092852
                     },
                     {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Register-0""
```

Comment:
Again, it shouldn't be necessary to disambiguate this, because the existing v1 `logicalLocation` has only a `name`, which is correct.